### ISSUE TITLE
Exporting data from inputs

### ISSUE THREAD

**Issue by grizzm0** [clarketm/TableExport#102]
We've got a table where some td's contains text while others contain an input. Inputs seems to be ignored by tableexport. Any plans to support this behaviour?

Example:
```html
<td>
  <input type="text">
</td>
```

**Comment by clarketm** [comment]
@grizzm0 – we are merely retrieving the cell's `textContent` at the moment. Albeit, it might be worthwhile to provide a `format` override that can be attached to the `td` element (or globally) to alter this behavior. I can provide that hook and prepare it for an upcoming release.

**Comment by bhansson** [comment]
It would be great if this was doable.

**Comment by bhansson** [comment]
Here is a working solution we're using at the moment, if there is an input, we use that value, if not we use textContent.
var content = val.getElementsByTagName('INPUT')[0] !== undefined ? val.getElementsByTagName('INPUT')[0].value : val.textContent;
return settings.formatValue(content);

**Comment by lfernandes** [comment]
> Here is a working solution we're using at the moment, if there is an input, we use that value, if not we use textContent.
> 
> ```js
> var content = val.getElementsByTagName('INPUT')[0] !== undefined ? val.getElementsByTagName('INPUT')[0].value : val.textContent;
> return settings.formatValue(content);
> ```

sorry, where you've inserted it?

**Comment by fuzunspm** [comment]
Yeah i need to know where to use `settings.formatValue(content);` property
